src/NServiceBus.Metrics.PerformanceCounters/SLA/EstimatedTimeToSLABreachCounter.cs

```diff
@@ -38,7 +37,7 @@ public void Update(DateTime sent, DateTime processingStarted, DateTime processin
 
     public void Start()
     {
-        timer = new Timer(RemoveOldDataPoints, null, 0, 2000);
+        timer = new System.Threading.Timer(RemoveOldDataPoints, null, 0, 2000);
```

Comment:
This is here because of our own Timer struct. Should we change the struct name or the namespace of the struct?